homeassistant/components/updater/__init__.py

```diff
@@ -52,30 +53,27 @@ def __init__(self, update_available: bool, newest_version: str, release_notes: s
 
 async def async_setup(hass, config):
     """Set up the updater component."""
-    if "dev" in current_version:
-        # This component only makes sense in release versions
-        _LOGGER.info("Running on 'dev', only analytics will be submitted")
-
     conf = config.get(DOMAIN, {})
-    if conf.get(CONF_REPORTING):
-        huuid = await hass.helpers.instance_id.async_get()
-    else:
-        huuid = None
 
-    include_components = conf.get(CONF_COMPONENT_REPORTING)
+    for option in (CONF_COMPONENT_REPORTING, CONF_REPORTING):
+        if option in conf:
+            _LOGGER.warning(
+                "Analytics reporting with the option '%s' "
+                "is deprecated and you should remove that from your configuration. "
+                "The analytics part of this integration has moved to the new 'analytics' integration",
+                option,
+            )
```

Comment:
Why not us `cv.deprecated` in the config schema?

Reply:
I want to mention the key, and does not care about it's value

Reply:
`cv.deprecated` mentions the key and doesn't care about it's value?

Reply:
Right... I was thinking of something else!
The response for `cv.deprecated` == I want to guide them to the analytics integration.

Reply:
Oh right, these users won't have the `default_config` 👍